example/transform/openai_model.ipynb

```diff
@@ -120,7 +120,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "raw_context_input = [\"It was a sunny day and the sky color is blue.\", \"My name is bobby and I am a talent software engineer working on AI/ML.\"]"
+    "raw_context_input = [\"\"\"It was a sunny day and the sky color is blue.\", \"My name is bobby and I am a talent software engineer working on AI/ML.\"\"\"]"
```

Comment:
Triple quotes and single quotes don't match.

Reply:
fixed